crates/ruma-events/src/space_order.rs

```diff
@@ -0,0 +1,79 @@
+//! Types for the [`m.space_order`] event.
+//!
+//! [`m.space_order`]: https://github.com/matrix-org/matrix-spec-proposals/pull/3230
+
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+/// The content of an `m.space_order` event.
+///
+/// Whether the space has been explicitly ordered.
+///
+/// This event appears in the user's room account data for the space room in
+/// question.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
+#[ruma_event(type = "m.space_order", kind = RoomAccountData)]
+pub struct SpaceOrderEventContent {
+    /// The current space order.
+    pub order: String,
+}
+
+impl SpaceOrderEventContent {
+    /// Creates a new `SpaceOrderEventContent` with the given value.
+    pub fn new(order: String) -> Self {
+        Self { order }
+    }
+}
+
+#[cfg(all(test, feature = "unstable-msc3230"))]
```

Comment:
It's not necessary to put the tests behind the cargo feature, the whole module should be behind it.

```suggestion
#[cfg(test)]
```